recipes/usgscsm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "usgscsm" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3a9617288cf70a0d3752a1e81eec9adacabf59092f2069e8635f5d91ce77d9d1
+
+build:
+  number: 0 
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [linux]
+    - cmake >=3.10
+  host:
+    - csm
+    - vc 14  # [win]
+  run:
+    - csm
+    - vc  # [win]
+
+test:
+  commands:
+    - conda inspect linkages -p $PREFIX $PKG_NAME  # [not win]
+    #- test -e $PREFIX/lib/libusgscsm.so  # [linux]
+    - test -e $PREFIX/lib/libusgscsm.dylib  # [osx]
```

Comment:
To be more generic you can do something like:
```
- test -e $PREFIX/lib/libusgscsm${SHLIB_EXT}  # [unix]
```

Reply:
Making this change is resulting in a build failure on azure.

Here is the environment location: `/home/conda/feedstock_root/build_artifacts/usgscsm_1552680604371/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_`

and here is the test locations:

`/home/conda/feedstock_root/build_artifacts/usgscsm_1552680604371/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_/lib/`

which looks like this should be working, but I am clearly missing something. My previous 'fix' on this was to check the linkage and then uncomment the file check - obviously not ideal. Any thoughts?